Phonebook.Frontend/src/app/shared/components/user/user-list-card/user-list-card.component.html

```diff
@@ -0,0 +1,71 @@
+<mat-card [routerLink]="['/user', person.Id]" class="mat-ripple list-card">
+    <mat-card-header *ngIf="actionButtonIcon">
+        <div class="pb-right-top-action">
+            <button propagationStop mat-icon.button [ngClass]="actionButtonClasses" (click)="buttonClicked()" class="close">
+                <mat-icon>{{ actionButtonIcon }}</mat-icon>
+            </button>
+        </div>
+    </mat-card-header>
+    <mat-card-content>
```

Comment:
can we also add the room of the user? That would be create.

Reply:
great ** i think he means great or @paule96  do u want to create something we dont know about yet ?